### ISSUE TITLE
readthedocs.io

### ISSUE THREAD

**Issue by pavel-lexyr** [easylist/easylist#8678]
### List the website(s) you're having issues:
`https://docs.readthedocs.io/en/stable/`

### What happens?
Ads appear

### List Subscriptions you're using:

Defaults

### Your settings

- OS/version: Linux
- Browser/version: Firefox (latest)
- Adblock Extension/version: AdNauseam(uBlock Origin), Latest

### Other details:

![image](https://user-images.githubusercontent.com/69010336/129800372-7aa00c7e-4c0c-4e0d-9449-b96e183f4a92.png)

Proposed change:
```
readthedocs.io##.ethical-rtd
readthedocs.io##.keep-us-sustainable
```

**Comment by pavel-lexyr** [comment]
It seems to have been partially fixed in #983.

`https://easylist.to/easylist/easylist.txt`, the file AdNauseam uses as its upstream, does not have it, though.

Is the file obsolete?

**Comment by ryanbr** [comment]
Check with uBO, this should be blocked already.
